sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1990,6 +1990,12 @@ code `invalid` if a user tries to add them to a graph.
 Removing this restriction is something we may look at for future revisions of
 `sycl_ext_oneapi_graph`.
 
+==== sycl_ext_oneapi_enqueue_functions
+
+The command submission commands defined in
```

Comment:
Nit but "command submission commands" is a little confusing, is there a better way to phrase it?

The extension refers to them collectively as functions so perhaps "command submission functions"?